recipe/meta.yaml

```diff
@@ -10,8 +10,16 @@ source:
   sha256: 9e8a7640911481134e60987d5d1e1cd157f430c3b4b38de8d36fc55c302bc299
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install .
+  number: 2
+  string: {{ impl }}_py{{ PY_VER }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  script:
+    - set "CMAKE_ARGS=%CMAKE_ARGS% -DUSE_CUDA=1"  # [impl == "cuda" and win]
+    - set "CMAKE_ARGS=%CMAKE_ARGS% -DUSE_GPU=1"   # [impl == "cpu" and win]
```

Comment:
Is this okay? I've changed the default CPU builds to include OpenCL as well since it's a lightweight dependency.

Reply:
Fine with me 👍